lib/ruby-debug-ide/attach/util.rb

```diff
@@ -65,7 +66,29 @@ def get_child_pids(pid)
     pipe.readlines.each do |child|
       child_pid = child.strip.to_i
       q.push(child_pid)
-      pids << child_pid
+      pids.add(child_pid)
+    end
+  end
+
+  filter_ruby_processes(pids)
+end
+
+def filter_ruby_processes(pids)
+  pipe = IO.popen(%Q(lsof -c ruby | awk '{print $2 ":" $9}' | grep -E 'bin/ruby([[:digit:]]+\.?)*$'))
+
+  ruby_processes = Set.new
+
+  pipe.readlines.each do |process|
+    pid = process.split(/:/).first
+    ruby_processes.add(pid.to_i)
+  end
+
+  pids.each do |pid|
+    if(!ruby_processes.include? pid)
+      $stderr.puts "Process with pid:#{pid} was filtered, because is not a ruby process.\n"
```

Comment:
Is it?
https://github.com/ruby-debug/ruby-debug-ide/blob/master/bin/gdb_wrapper#L93